### ISSUE TITLE
[BUG]BetterPlayerTheme.custom controls are always visible

### ISSUE THREAD

**Issue by Farg0k** [jhomlala/betterplayer#1045]
**History check**
yes

**Describe the bug**
When using BetterPlayerTheme.custom controls are always visible (dont work controlsHideTime) and dont work setControlsVisibility(false) 

**To Reproduce**
1. Go to example app 
2. Click on 'Change player theme'
3. Select 'Custom'
4. See controls are always visible

**Example code*
code not required

**Expected behavior**
Expected to hide the controls  as in the material or cupertino theme and the reaction to use setControlsVisibility(false) 

**Screenshots**
not informative

**Flutter doctor**
[√] Flutter (Channel stable, 3.0.3, on Microsoft Windows [Version 10.0.19043.1766], locale ru-UA)
[√] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.1.3)
[√] Android Studio (version 2021.2)
[√] VS Code (version 1.68.0)
[√] Connected device (4 available)
[√] HTTP Host Availability

• No issues found!

**Better Player version**
- Version: 0.0.83

**Smartphone (please complete the following information):**
 - Device: [AWD Pixel 2]
 - OS: [Android 11 x86]

**Additional context**
-

**Comment by Tiagocf2** [comment]
Any updates? Still having trouble to hide/show custom controls

**Comment by Farg0k** [comment]
Yes. I solved this by writing my own interface which is visible when needed.

**Comment by darter0** [comment]
can you share it please? i'm having the same issue.

**Comment by Farg0k** [comment]
I created custom controls to control the player via a remote control. I don't think it will work for you.

**Comment by jhomlala** [comment]
After reviewing this, the described behavior is working as intended. When you use the custom theme (`PlayerTheme.custom`), the library intentionally delegates 100% of the UI rendering and logic to your `customControlsBuilder`. It is completely up to your custom widget to listen to the player events, handle the `controlsHideTime` timer, and manage its own visibility or animations. The `CustomControlsWidget` in the example app is just a basic, static demonstration of how to hook up a custom layout and doesn't implement auto-hide logic. You can look at how `BetterPlayerMaterialControls` is implemented if you want to see how to handle timers and visibility. Closing this issue — feel free to re-open if you have further questions.
